test/e2e/edge-can-read-request-body/app/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dependencies": {
+    "next": "canary",
+    "react": "latest",
+    "react-dom": "latest"
+  }
+}
```

Comment:
I think the e2e tests don't define dependencies so it can use the version defined in this repo